src/test/java/guiTest/LoginTest.java

```diff
@@ -0,0 +1,118 @@
+package guiTest;
+
+import org.junit.Before;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+import pages.Login;
+import utils.FunctionalTest;
+
+import static org.junit.Assert.*;
+import static utils.Constants.*;
+
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+
+public class LoginTest extends FunctionalTest {
+
+    Login login = new Login(driver);
+
+    @Before
+    public void init() {
+        driver.get(LOGIN_URL);
+    }
+
+    @Test
+    public void test01_noData_LoginButtonIsDisabled() {
+        assertFalse(login.isLoginButtonEnabled());
+    }
+
```

Comment:
Should be a part of test06

Reply:
The test01 is verify that the button is enabled or disabled with no entered data(email).